### PR TITLE
feat: add docs about limiting API access by IP

### PR DESCRIPTION
**Pull request by jfhinchcliffe**
we've just added this feature to the UI / Graphql, so just getting the docs site in order 😁

The `http://localhost:3000/docs/apis/managing-api-tokens` page seems like the most logical location.
https://bk-docs-pr-1924.onrender.com/docs/apis/managing-api-tokens#limiting-api-access-by-ip-address

### CODE REVIEW

**Comment by jfhinchcliffe** [comment]
Waiting until the next GraphQL Schema Update goes in, so I can directly reference the new mutation.

File: pages/apis/managing_api_tokens.md [hunk 1]
```diff
@@ -60,6 +60,14 @@ Removing access from a token will send a notification email to the token's owner
 >📘
 > Removing access from a token does not delete the token, however token owner cannot re-add your organization to their token's scope.
 
+## Limiting API Access by IP address
```
Comment (mbelton-buildkite) [inline pages/apis/managing_api_tokens.md:63]: Nitpick: Add a blank line to separate elements like headings and paragraphs.

```suggestion
## Limiting API Access by IP address

```

File: pages/apis/managing_api_tokens.md [hunk 2]
```diff
@@ -60,6 +60,14 @@ Removing access from a token will send a notification email to the token's owner
...
+>🚧 Beta Feature
+> This feature is not yet available to all organizations.
+
+If you'd like to limit access to your organization by IP address, you can create an allow list of IP addresses on the [organization's member permissions page](https://buildkite.com/organizations/~/member-permissions).
```
Comment (mbelton-buildkite) [inline pages/apis/managing_api_tokens.md:67]: - "Allowlist" is one word.
- TIL that you can format links like that to autocomplete the org in the URL. 🔥
- I've tried a different way of describing the location that I think is clearer, but let me know what you think.

```suggestion
If you'd like to limit access to your organization by IP address, you can create an allowlist of IP addresses in the [organization's permission settings](https://buildkite.com/organizations/~/member-permissions).
```

File: pages/apis/managing_api_tokens.md [hunk 3]
```diff
@@ -60,6 +60,14 @@ Removing access from a token will send a notification email to the token's owner
...
+
+The allow list can also be managed using the [GraphQL API](/docs/apis/graphql-api) with the `organizationAPIIPAllowListUpdate` mutation.
```
Comment (mbelton-buildkite) [inline pages/apis/managing_api_tokens.md:69]: Nitpicks:

- allow list -> allowlist
- Passive voice -> active
- Link to the specific mutation (I assume that's the one you were waiting for)

```suggestion
You can also manage the allowlist with the [`organizationAPIIPAllowListUpdate`](/docs/apis/graphql/schemas/mutation/organizationapiipallowlistupdate) mutation in the GraphQL API.
```

Comment (mbelton-buildkite) [reply pages/apis/managing_api_tokens.md:69]: Is it too late to rename the mutation? I think it should be `organizationApiIpAllowlistUpdate `

File: pages/apis/managing_api_tokens.md [hunk 4]
```diff
@@ -60,6 +60,14 @@ Removing access from a token will send a notification email to the token's owner
 >📘
 > Removing access from a token does not delete the token, however token owner cannot re-add your organization to their token's scope.
 
+## Limiting API Access by IP address
+>🚧 Beta Feature
+> This feature is not yet available to all organizations.
```
Comment (mbelton-buildkite) [inline pages/apis/managing_api_tokens.md:65]: I'll follow up about this part in a DM

**Review by mbelton-buildkite — COMMENTED** [review]
Nice! I've added a few thoughts to consider 😊

File: pages/apis/managing_api_tokens.md (hunk 2 above)
Comment (jameshill) [reply pages/apis/managing_api_tokens.md:67]: > "Allowlist" is one word.

TIL... i've been writing it as Allow list

File: pages/apis/managing_api_tokens.md (hunk 3 above)
Comment (jfhinchcliffe) [reply pages/apis/managing_api_tokens.md:70]: Nope! PR here: https://github.com/buildkite/buildkite/pull/11543

File: pages/apis/managing_api_tokens.md (hunk 4 above)
Comment (jameshill) [reply pages/apis/managing_api_tokens.md:66]: Hey @jfhinchcliffe (@mbelton-buildkite)

Yeah, I was just trying to be consistent across the board with the communication style.

My concern primarily stems from the GraphQL implementation. 

Given that this mutation becomes public knowledge and discoverable via the GraphQL explorer or introspection queries. 

If it _looks_ to be functional, but isn't functional due to the feature flag then I think we have a problem. Especially when we're dealing with security related features like an Allowlist. 

However, I'm happy to defer to @mbelton-buildkite for best practices here.

File: pages/apis/managing_api_tokens.md (hunk 3 above)
Comment (jfhinchcliffe) [reply pages/apis/managing_api_tokens.md]: Above PR merged. Updated to `organizationApiIpAllowlistUpdate` here.

File: pages/apis/managing_api_tokens.md (hunk 4 above)
Comment (jfhinchcliffe) [reply pages/apis/managing_api_tokens.md]: changed to 'Private Beta' and updated screenshot above 😁

Comment (jfhinchcliffe) [reply pages/apis/managing_api_tokens.md]: Removed 'Beta' wording.

**Comment by jfhinchcliffe** [comment]
heya @mbelton-buildkite This feature is live on prod. Is this docs change good to merge?

**Comment by mbelton-buildkite** [comment]
@jfhinchcliffe, I had a chat with @jameshill about this and we agreed that the docs should only contain features at least in the Beta release phase. I believe this is in Private beta, so it shouldn't be merged yet.

**Comment by jameshill** [comment]
> @jfhinchcliffe, I had a chat with @jameshill about this and we agreed that the docs should only contain features at least in the Beta release phase. I believe this is in Private beta, so it shouldn't be merged yet.

Hey @mbelton-buildkite this feature is now GA.

So if you're happy with the documentation... we're good to launch.

**Review by mbelton-buildkite — APPROVED** [review]
Great news that this is GA. Looks good to go :)

**Pull request merged**